sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -18,88 +18,20 @@ import org.json.JSONObject
  * @property method The [RequestMethod] will determine the type of request being made
  * @property queuedTime The wall clock time that the request was enqueued
  * @property uuid unique identifier of this request
+ *
+ * @constructor
  */
 internal open class KlaviyoApiRequest(
     val urlPath: String,
     val method: RequestMethod,
-    override val queuedTime: Long = Registry.clock.currentTimeMillis(),
-    override val uuid: String = UUID.randomUUID().toString()
+    queuedTime: Long? = null,
+    uuid: String? = null
 ) : ApiRequest {
 
     internal enum class Status {
         Unsent, Inflight, PendingRetry, Complete, Failed
     }
 
-    override val type: String = urlPath
```

Comment:
I did a bit of file organizing, usually I order it like this:
Subtypes
Companion objects
Properties
Constructors
Methods

Things had gotten disorganized here